pulsar-client-cpp/tests/standalone.conf

```diff
@@ -28,10 +28,10 @@ globalZookeeperServers=
 # Configuration Store connection string
 configurationStoreServers=
 
-brokerServicePort=8885
+brokerServicePort=6650
```

Comment:
This shouldn't be changed

Reply:
Then maybe we should remove this line in the readme,
`export PULSAR_STANDALONE_CONF=${PULSAR_PATH}/pulsar-client-cpp/tests/standalone.conf
`As it will make standalone server start at 8885, but all app test will try to connect server at 6650 and fail.

Reply:
@MarvinCai This 8885 is for cpp tests only. and cpp test client also have 8885 configured.

Reply:
@jiazhai hmmmm, maybe I missed something, but in cpp BasicEndtoEndTests it's looking for pulsar service at 6650([link](https://github.com/apache/pulsar/blob/master/pulsar-client-cpp/tests/BasicEndToEndTest.cc#L49-L50)), and if I don't change this config most test case will fail.

Reply:
@MarvinCai  How do you run the test cases? usually we run it by `docker-tests.sh` under dir `pulsar-client-cpp`

Reply:
The tests should not be affected by changes in `README.md`

Reply:
Oh I was following instruction in README.md of manually start a standalone server then directly compile and run all tests, probably that's the reason. 
Got it, I'll just update the instruction on installing protobuf.